app/views/shared/_resources_table.html.haml

```diff
@@ -2,9 +2,21 @@
   %thead.thead-light
     %tr
       %th #
+      - exclude_columns = [:province, :actions, :level]
```

Comment:
`exclude_columns` option should be passed from the caller of this partial whenever they need it. This partial is used my many places, so we can't just hardcodes fields from a specific model.

`exclude_columns` option should be optional.